app/helpers/security_group_helper/textual_summary.rb

```diff
@@ -40,9 +40,9 @@ def textual_group_firewall
       [
         rule.network_protocol,
         rule.host_protocol,
-        rule.direction,
+        _(rule.direction),
         port_range_helper(rule),
-        (rule.source_ip_range || rule.source_security_group.try(:name) || "<None>")
+        (rule.source_ip_range || rule.source_security_group.try(:name) || _("<None>"))
```

Comment:
I'm fine withi his as is, but I wonder if it's better to do something like the following moving forward:

```suggestion
        (rule.source_ip_range || rule.source_security_group.try(:name) || "<" + _("None") + ">")
```